src/plugins/console/public/styles/_app.scss

```diff
@@ -33,6 +33,47 @@
 .conApp__output {
   display: flex;
   flex: 1 1 1px;
+
+  .ace_badge {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding: 0 8px;
+    display: inline-block;
+    text-decoration: none;
+    border-radius: 3px;
+    border: solid 1px transparent;
+    white-space: nowrap;
+    vertical-align: middle;
+    cursor: default;
+    max-width: 100%;
+    text-align: left;
+  }
+
+  .ace_badge-success {
+    background-color: $euiColorSuccess;
+    color: chooseLightOrDarkText($euiColorSuccess);
+  }
+
+  .ace_badge-primary {
+    background-color: $euiColorPrimary;
+    color: chooseLightOrDarkText($euiColorPrimary);
+  }
+
+  .ace_badge-default {
+    background-color: $euiColorGhost;
```

Comment:
just interesting why we need `chooseLightOrDarkText` for `color` and don't need it for `background-colo`r

Reply:
According to EUI, `chooseLightOrDarkText`is used  _to determine whether or not to use light (white) or dark (black) text against the given background color._ See https://elastic.github.io/eui/#/utilities/color-functions#light-or-dark-text